Assets/Scripts/EndGoal.cs

```diff
@@ -8,28 +8,34 @@ public class EndGoal : MonoBehaviour
 {
 
     private string frogsRemainingText;
-    private int frogGoal = 1;
-
-
-    void Update()
+    private int frogGoal = 0;
+    private int frogAmount = 0; //amount of frogs in level
+    public int FrogGoal 
     {
-        frogsRemainingText = $"Frogs: {frogGoal}/{FrogSpawner.amountOfFrogs}";
+            get { return frogGoal; }
     }
 
+    private ParticleSystem goalParticles; //confetti particle system
+
     //observer pattern that tells frog remaining UI how many frogs left
-    public static event Action<string> FrogReachesGoalTriggered; 
+    public static event Action<string> FrogReachesGoalTriggered;
```

Comment:
What is the actual event that happens here? The Frog has reached the goal? I think it's overkill to name it GoalReached and Triggered. I'd probably just go with `GoalReached` if that's accurate.